packages/victory-candlestick/src/helper-methods.js

```diff
@@ -76,34 +113,131 @@ const getDataStyles = (datum, style, props) => {
   return assign({}, style, { stroke, fill });
 };
 
-const getLabelProps = (dataProps, text, style) => {
-  const { x, high, index, scale, datum, data, horizontal } = dataProps;
-  const labelStyle = style.labels || {};
-  const defaultAnchors = {
-    vertical: horizontal ? "middle" : "end",
-    text: horizontal ? "start" : "middle"
+const getText = (props, type) => {
+  const { datum, index, labels } = props;
+  const propName = `${type}Labels`;
+  const labelProp = props[propName];
+  if (!labelProp && !labels) {
+    return null;
+  } else if (labelProp === true || labels === true) {
+    const dataName = `_${type}`;
+    return `${datum[dataName]}`;
+  }
+  return Array.isArray(labelProp) ? labelProp[index] : labelProp;
+};
+
+const getCandleWidth = (props, style) => {
+  const { active, datum, data, candleWidth, scale, defaultCandleWidth } = props;
```

Comment:
This is a little confusing. You're using the same function that was in `Candle` and supplying it with the data props that you're calculating for each `Candle`. That mostly works, but `active` will not typically be defined at this level. Typically it would get added to state by Victory's event system.